conduct/code-of-conduct.md

```diff
@@ -1,17 +1,21 @@
 # Code of Conduct
 
-R-Ladies is dedicated to providing a harassment-free experience for everyone. We do not tolerate harassment of participants in any form.
+`<EVENT>` is dedicated to providing a harassment-free experience for participants at our event(s). `<EVENT>` is a working conference intended for professional networking and collaboration in the R community. They exist to encourage the open exchange of ideas and expression and require an environment that recognizes the inherent worth of every person and group. 
```

Comment:
I think that the last sentence belongs in the next paragraph.

Reply:
Minor thing - but would it be clearer to say only "conference" instead of "working conference"?